sherpa/astro/ui/utils.py

```diff
@@ -718,6 +718,168 @@ def show_bkg_model(self, id=None, bkg_id=None, outfile=None, clobber=False):
         txt = self._get_show_bkg_model(id, bkg_id)
         send_to_pager(txt, outfile, clobber)
 
+    def calc_bkg_stat(self, id=None, *otherids):
+        """Calculate the fit statistic for a background data set.
+
+        Evaluate the current background models for the background
+        datasets, calculate the statistic for each background, and
+        return the sum.  No fitting is done, as the current model
+        parameter, and any filters, are used. The `calc_bkg_stat_info`
+        routine should be used if the result for a particular
+        background component needs to be returned.
+
+        .. versionadded:: 4.16.0
+
+        Parameters
+        ----------
+        id : int or str, optional
+           The data set that provides the data. If not given then all
+           background data sets with an associated background model
+           are used simultaneously.
+        *otherids : int or str, optional
+           Other data sets to use in the calculation.
+
+        Returns
+        -------
+        stat : number
+           The current statistic value.
+
+        See Also
+        --------
+        calc_bkg_stat_info, calc_stat, fit_bkg, get_bkg_stat_info,
+        set_stat
+
+        Examples
+        --------
+
+        Calculate the statistic for the background in the default data
+        set:
+
+        >>> stat = calc_bkg_stat()
+
+        Find the statistic for the background for data set 3:
+
+        >>> stat = calc_bkg_stat(3)
+
+        Calculate the background statistic value using two different
+        statistics:
+
+        >>> set_stat('chi2datavar')
+        >>> s1 = calc_bkg_stat()
+        >>> set_stat('chi2gehrels')
+        >>> s2 = calc_bkg_stat()
+
+        """
+        ids, f = self._get_bkg_fit(id, otherids)
+        return f.calc_stat()
+
+    def calc_bkg_stat_info(self):
+        """Display the statistic values for the current background models.
+
+        Returns the statistic values for the background datasets.  See
+        calc_stat_info.
```

Comment:
Suggest rewording with the direct link to calc_stat_info, or include the full information about the output.

Returns the statistics values for the current background dataset, as in `calc_stat_info` for the source dataset.

Reply:
I am reluctant to copy the documentation over from `calc_stat_info` as I can tell you from direct experience that it's hard to keep things up to date. I am not sure what you mean by "a direct link". We have:

- for the python user - calling `help(calc_bkg_stat_info)` - the "See Also" section a few lines below this line which already mentions `calc_stat_info`
- for the users of RTD this will become an actual link, which you can see an example of in the RTD build of this PR at https://sherpa--1891.org.readthedocs.build/en/1891/ui/api/sherpa.astro.ui.calc_bkg_stat_info.html
- for CIAO users I convert these files into ahelp files; technically there can be a link between the two but for various technical reasons it can get confusing - e.g. see the `calc_stat_info` "see also" section for [CIAO 4.15](https://cxc.harvard.edu/sherpa/ahelp/calc_stat_info.html) and it's not clear what the version for this command will look like

Reply:
I agree that copying is not a good solution and I thought that updating the wording to specifically say that this as in `calc_stat_info`. For me See Also does not necessarily mean that you are going to see the exact output of `calc_bkg_stat_info`

Reply:
That's good. Thanks